src/main/java/server/acode/domain/review/service/ReviewService.java

```diff
@@ -0,0 +1,192 @@
+                .orElseThrow(() -> new CustomException(ErrorCode.REVIEW_INTENSITY_NOT_FOUND));
+        String intensity = registerReviewRequest.getIntensity().toLowerCase();
+        System.out.println("intensity : " + intensity);
+        increaseReviewIntensity(reviewIntensity, intensity);
+
+        ReviewStyle reviewStyle = reviewStyleRepository.findByFragrance(fragrance)
+                .orElseThrow(() -> new CustomException(ErrorCode.REVIEW_STYLE_NOT_FOUND));
+        String styleList = registerReviewRequest.getStyle().toLowerCase();
+        System.out.println("styleList : " + styleList);
+        List<String> keywordList = Arrays.asList(styleList.split(", "));
+        System.out.println("keywordList : " + keywordList);
+        keywordList.forEach(keyword -> increaseReviewStyle(reviewStyle, keyword));
+
+        return ResponseEntity.status(HttpStatus.CREATED).build();
+    }
+
+    public void increaseReviewSeason(ReviewSeason reviewSeason, String season) {
+        try {
+            // ReviewSeason 클래스에서 지정된 계절에 해당하는 필드를 가져오기 위해 리플렉션 사용
+            // getDeclaredField 메소드 : 클래스의 모든 필드 중에서 지정된 이름의 필드 반환
+            Field field = ReviewSeason.class.getDeclaredField(season);
+
+            // 리플랙션 사용 시, 가져온 필드에 대한 접근 권한 설정
+            // setAccessible(true)로 private 필드 접근 가능
+            field.setAccessible(true);
+
+            // 가져온 필드의 현재 값 읽어오기 근데 정수로
+            int fieldValue = (int) field.get(reviewSeason);
+
+            // 필드 값 1 증가
+            field.set(reviewSeason, fieldValue + 1);
+
+            System.out.println("increaseReviewSeason");
+        } catch (NoSuchFieldException | IllegalAccessException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void increaseReviewLongevity(ReviewLongevity reviewLongevity, String longevity) {
+        try {
+            // 클래스에서 지정된 필드를 가져오기 위해 리플렉션 사용
+            // getDeclaredField 메소드 : 클래스의 모든 필드 중에서 지정된 이름의 필드 반환
+            Field field = ReviewLongevity.class.getDeclaredField(longevity);
+
+            // 리플랙션 사용 시, 가져온 필드에 대한 접근 권한 설정
+            // setAccessible(true)로 private 필드 접근 가능
+            field.setAccessible(true);
+
+            // 가져온 필드의 현재 값 읽어오기 근데 정수로
+            int fieldValue = (int) field.get(reviewLongevity);
+
+            // 필드 값 1 증가
+            field.set(reviewLongevity, fieldValue + 1);
+
+            System.out.println("increaseReviewLongevity");
+        } catch (NoSuchFieldException | IllegalAccessException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void increaseReviewIntensity(ReviewIntensity reviewIntensity, String intensity) {
+        try {
+            // 클래스에서 지정된 필드를 가져오기 위해 리플렉션 사용
+            // getDeclaredField 메소드 : 클래스의 모든 필드 중에서 지정된 이름의 필드 반환
+            Field field = ReviewIntensity.class.getDeclaredField(intensity);
+
+            // 리플랙션 사용 시, 가져온 필드에 대한 접근 권한 설정
+            // setAccessible(true)로 private 필드 접근 가능
+            field.setAccessible(true);
```

Comment:
외부에서 접근 권한 설정을 바꾸기보다 클래스 내부에서 String에 해당하는 필드 값을 1씩 증가시키는 메소드를 두고 호출하면 더 좋을 것 같다고 생각해봤어욤
```suggestion
            reviewLongevity.incrementIntensity(intensity);
```

찾아보니까 map을 사용해도 좋을 것 같아요!
```java
// 강도를 나타내는 문자열과 해당 필드를 매핑하는 맵
    private static final Map<String, Function<ReviewIntensity, Integer>> INTENSITY_FIELD_MAP;

    static {
        INTENSITY_FIELD_MAP = Map.of(
                "weak", ReviewIntensity::getWeak,
                "medium", ReviewIntensity::getMedium,
                "strong", ReviewIntensity::getStrong,
                "intense", ReviewIntensity::getIntense
        );
    }

    // 문자열에 해당하는 필드 값을 1씩 증가시키는 메소드
    public void incrementIntensity(String intensity) {
        INTENSITY_FIELD_MAP.getOrDefault(intensity.toLowerCase(), field -> {
            // 예외 처리
            throw new IllegalArgumentException();
        }).apply(this); // 해당 강도에 매핑된 필드의 값을 1 증가
    }
```

Reply:
아하 조금 더 공부해보고 적용하도록 하겠습니닷🤓🤓